tests/test_libzim_creator.py

```diff
@@ -614,6 +614,10 @@ def test_hints_values(fpath):
         c.add_redirection(
             path="4", title="", targetPath="0", hints={Hint.COMPRESS: True}
         )
+        # filtered-out values
+        c.add_item(StaticItem(path="5", title="", hints={5: True}))
```

Comment:
Would be nice to also test with the int value corresponding to the real value of the hint key.
(Should be accept it or filter it ?)